lib/util.js

```diff
@@ -316,7 +319,7 @@ function formatValue(ctx, value, recurseTimes) {
     formatter = formatMap;
   } else {
     // Only create a mirror if the object superficially looks like a Promise.
```

Comment:
Yes, now the check isn't superficial. Also the comment isn't in the right context. It is not obvious here that we create an object mirror.
I know the change was made earlier but do you mind moving it to the `inspectPromise` declaration ?